models/brain_image_synthesis_latent_diffusion_model/scripts/saver.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+import nibabel as nib
+import numpy as np
+import torch
+
+
+class NiftiSaver:
+    def __init__(self, output_dir: str) -> None:
+        super().__init__()
+        self.output_dir = output_dir
+        self.affine = np.array(
```

Comment:
The affine is harded-coded for the data this bundle used?

Reply:
This was eliminated by using `SaveImage`.